openml/study/functions.py

```diff
@@ -386,7 +386,7 @@ def detach_from_suite(suite_id: int, task_ids: List[int]) -> int:
         OpenML id of the study
 
     task_ids : list (int)
-        List of entities to link to the collection
+        List of entities to  de-link (detach) from the collection
```

Comment:
I think just using "detach" is sufficient. If you want to keep both, I believe 'delink' is a single word, but 'unlink' is probably preferred. (also goes for L#407 below )